mm/2s2h/Rando/Logic/Regions/MilkRoad.cpp

```diff
@@ -94,22 +99,32 @@ static RegisterShipInitFunc initFunc([]() {
         },
         .exits = { //     TO                                         FROM
             EXIT(ENTRANCE(TERMINA_FIELD, 5),                ENTRANCE(MILK_ROAD, 0), true),
-            EXIT(ENTRANCE(ROMANI_RANCH, 0),                 ENTRANCE(MILK_ROAD, 1), true),
-            EXIT(ENTRANCE(GORMAN_TRACK, 3),                 ENTRANCE(MILK_ROAD, 2), RANDO_EVENTS[RE_COWS_FROM_ALIENS]),
+            EXIT(ENTRANCE(ROMANI_RANCH, 0),                 ENTRANCE(MILK_ROAD, 1), AFTER(TIME_DAY3_AM_06_00) || (HAS_ITEM(ITEM_POWDER_KEG) && CAN_BE_GORON)),
             EXIT(ENTRANCE(GORMAN_TRACK, 0),                 ENTRANCE(MILK_ROAD, 3), true),
         },
+        .connections = {
+            CONNECTION(RR_MILK_ROAD_BEHIND_FENCE, (CAN_BE_GORON && HAS_ITEM(ITEM_BOMB)) || (RANDO_EVENTS[RE_COWS_FROM_ALIENS] && IS_NIGHT2()) || FINAL_DAY()),
```

Comment:
We only have glitchless logic for now, so Goron bomb shouldn't be considered until we implement glitches/tricks, which is beyond the scope of this PR. Leaving a TODO for the future is fine.
```suggestion
            // TODO: Trick to Goron bomb jump over the fence
            CONNECTION(RR_MILK_ROAD_BEHIND_FENCE, (RANDO_EVENTS[RE_COWS_FROM_ALIENS] && IS_NIGHT2()) || FINAL_DAY()),
```